examples/jenkins/README.md

```diff
@@ -73,6 +73,43 @@ Steps
 
         $ oc get services -n test | grep frontend
 
+Advanced
+-----
+
+You can also work with this sample and demonstrate the use of the [kubernetes-plugin](https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin) to manage
+Jenkins slaves that run as on-demand Pods.  The kubenetes-plugin is pre-installed into the OpenShift Jenkins Images
+for Centos and RHEL produced by the [OpenShift Jenkins repository](https://github.com/openshift/jenkins).  The OpenShift
+Jenkins repository also produces a [base Jenkins slave image](https://github.com/openshift/jenkins/tree/master/slave-base),
+as well as sample Jenkins slave images for [Maven](https://github.com/openshift/jenkins/tree/master/slave-maven) and
+[NodeJS](https://github.com/openshift/jenkins/tree/master/slave-nodejs) which extend that base Jenkins slave image.
```

Comment:
I don't think we should refer to these as "sample" images. We (Red Hat) ship these images, and as such they should be supported. 

I also feel that telling customer that they should use these images as "samples" to build other images, portrays a very unpolished look and feel to the product. We should ship images for all of the supported SCL images, if we are going to promote this route.